src/http/any-catchall/_backfill-params.mjs

```diff
@@ -15,19 +15,15 @@ export default function backfill (basePath, apiPath, pagePath, req) {
   tmpl = tmpl.replace(base, '')
     .replace(/index\.mjs|\.mjs/, '')
     .replace(/(\/?)\$\$\/?$/, '$1(.*)')
-    .replace('$', ':')
+    .replaceAll(/\/\$(\w+)/g, "/:$1")
```

Comment:
We can't use `replaceAll` as we support Node 14.x in Lambda. Even though we recommend a development environment of Node 16.x we still support Node 14.x as a runtime. See: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

Reply:
Didn't I address this already in b2123f11 ?

Reply:
I pulled your changes before you made the switch. 😊